apps/backend-docker/.env.example

```
@@ -13,3 +13,13 @@ HATCHET_CLIENT_TOKEN=__HATCHET_CLIENT_TOKEN__
 HATCHET_CLIENT_HOST_PORT=localhost:7077
 HATCHET_CLIENT_TLS_STRATEGY=none
 HATCHET_LOG_LEVEL=DEBUG
+
+# Canonical app origins (prefer these; issuers fall back to them)
+APP_ORIGIN_AUTH="http://127.0.0.1:3010"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)